data_hub_api/docmaps/v2/sql/docmaps_index.sql

```diff
@@ -353,7 +353,13 @@ t_result_with_sorted_manuscript_versions_array AS (
         result.senior_editor_details,
         result.author_names_csv,
         result.evaluations,
-        CONCAT(publication.publication_date, ' ', publication.utc_publication_time) AS rp_publication_timestamp,
+        FORMAT_TIMESTAMP(
```

Comment:
If you format it, then the field should be called `rp_publication_timestamp_str`. But why not return the timestamp to Python?

Reply:
Maybe we can look into it together as I dont get what you mean.

Reply:
Sure.

But `PARSE_TIMESTAMP` returns a timestamp. That is what I would expect.
`FORMAT_TIMESTAMP` then turns it back to a string. That is what I would remove. Then `rp_publication_timestamp` would indeed be a timestamp. We can use `isoformat` to format it within Python where needed.